pkg/apis/autoscaling/annotation_validation.go

```diff
@@ -105,6 +110,22 @@ func validateFloats(annotations map[string]string) *apis.FieldError {
 	return errs
 }
 
+func validateScaleDownDelay(annotations map[string]string) *apis.FieldError {
+	var errs *apis.FieldError
```

Comment:
```suggestion
func validateScaleDownDelay(annotations map[string]string) (errs *apis.FieldError) {
```
A rare case when I actually prefer the named return value :)
Though in this particular case I'd restructure differently -- see suggestions below.

Reply:
this is just copy-pastaed from the other validates, so we might want to change them all in one go separately

Reply:
(unless you think we should change just this one?)